examples/libf3d/python/render_image.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import f3d
+
+if __name__ == "__main__":
+    if len(sys.argv) > 4:
+        sys.exit(1)
+
+    try:
+        f3d.Engine.autoload_plugins()
+
+        eng = f3d.Engine(f3d.Window.NATIVE_OFFSCREEN)
+        eng.loader.load_geometry(sys.argv[1])
+
+        img = eng.window.setSize(300, 300).renderToImage()
```

Comment:
```suggestion
        eng.window.size = 300, 300
        img = eng.render_to_image()
```
C++ style `setSize` and `renderToImage` are deprecated